README.md

```diff
@@ -168,7 +168,9 @@ Having a route with multiple parameters may affect negatively the performance, s
 
 <a name="match-order"></a>
 ##### Match order
-The routes are matched in the following order:
+The routing algorithm tokenizes the request path into segments.
```

Comment:
(divided by /).

Reply:
=> `The routing algorithm tokenizes (use the '/' to split) the request path into segments.`??

Reply:
not perfect but it’ll work.